cas-server-core/src/main/java/org/jasig/cas/logout/LogoutManagerImpl.java

```diff
@@ -118,15 +118,7 @@ public void setIssueAsynchronousCallbacks(final boolean asyncCallbacks) {
      */
     @Override
     public List<LogoutRequest> performLogout(final TicketGrantingTicket ticket) {
-        final Map<String, Service> services;
-        // synchronize the retrieval of the services and their cleaning for the TGT
-        // to avoid concurrent logout mess ups
-        synchronized (ticket) {
-            services = ticket.getServices();
-            ticket.removeAllServices();
-        }
-        ticket.markTicketExpired();
-
+        final Map<String, Service> services = ticket.getServices();
```

Comment:
Appreciate the comment. Would you mind turning this into a Java comment, or better yet, push a commit that turns that into a javadoc?